FeedStoreChallenge/CoreDataModel/ManagedFeedImage+CoreDataProperties.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  ManagedFeedImage+CoreDataProperties.swift
+//  FeedStoreChallenge
+//
+//  Created by Fei Wen on 6/5/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+//
+
+import Foundation
+import CoreData
+
+extension ManagedFeedImage {
+	@nonobjc public class func fetchRequest() -> NSFetchRequest<ManagedFeedImage> {
+		return NSFetchRequest<ManagedFeedImage>(entityName: "ManagedFeedImage")
+	}
+
+	@NSManaged public var id: UUID
+	@NSManaged public var imageDescription: String?
+	@NSManaged public var location: String?
+	@NSManaged public var url: URL
+	@NSManaged public var cache: ManagedCache
```

Comment:
`ManagedFeedImage` is an implementation detail of `CoreDataFeedStore` thus its properties should not be public.
```suggestion
	@NSManaged var id: UUID
	@NSManaged var imageDescription: String?
	@NSManaged var location: String?
	@NSManaged var url: URL
	@NSManaged var cache: ManagedCache
```